src/main/resources/db/migration/V20170713134558340__create_item_sequence.sql

```diff
@@ -0,0 +1,24 @@
+DELIMITER //
+CREATE PROCEDURE CreateSequence (`name` VARCHAR(30), `start` INT, `inc` INT)
+  BEGIN
+    CREATE TABLE IF NOT EXISTS `_sequences`
+    (
+      `name` VARCHAR(70) NOT NULL UNIQUE,
+      `next` INT NOT NULL,
+      `inc` INT NOT NULL
+    );
+
+    INSERT INTO `_sequences` VALUES (name, start, inc);
+  END //
+DELIMITER;
+
+CALL CreateSequence('item_id_seq', 1, 1);
+
+DELIMITER //
+CREATE FUNCTION NextVal (`vname` VARCHAR(30))
+  RETURNS INT DETERMINISTIC
```

Comment:
This all had to do with AWS RDS and how it does not allow the same things I was able to do locally.  I was able to get it to work with parameter groups and selecting something to allow the creation of the func.  I can try it without deterministic to see if that will work now.  Possibly since making the param group change I am not able to drop the deterministic part.